### PR TITLE
Switch ONNX to be default path

### PR DESCRIPTION
**Pull request by mbencer**
Ticket: 75535

### CODE REVIEW

**Review by tomdol — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Review by jane-intel — APPROVED** [review]

File: tests/layer_tests/onnx_tests/test_loop.py [hunk 1]
```diff
@@ -279,7 +279,6 @@ def test_loop_simple_precommit(self, ie_device, precision, ir_version, temp_dir)
     @pytest.mark.precommit
     @pytest.mark.timeout(250)
     def test_loop_in_loop_simple_precommit(self, ie_device, precision, ir_version, temp_dir):
-        if ie_device == 'GPU':
-            pytest.skip('Loop not supported on GPU')
+        pytest.skip('The model used in the test is incorrect according to ONNX standart: 70158')
         self._test(*self.create_loop_in_loop(), ie_device, precision, ir_version, temp_dir=temp_dir,
```
Comment (rkazants) [inline tests/layer_tests/onnx_tests/test_loop.py:283]: it is sad that layer tests no longer operate for legacy MO isn't it?

Comment (mbencer) [reply tests/layer_tests/onnx_tests/test_loop.py:283]: There is a problem with a model used for this test (it seems to be incorrect).
Even `onnx.checker.check_model` shows that the model is invalid.
As part of 70158 ticket we have a plan to fix the used model and restore the test

**Comment by rkazants** [comment]
guys, let's check how can we proceed without validation (layer tests, e2e) of legacy MO ONNX if we still have some models that require legacy path.

**Review by rkazants — APPROVED** [review]

**Comment by rkazants** [comment]
> guys, let's check how can we proceed without validation (layer tests) of legacy MO ONNX if we still have some models that require legacy path.

Will discuss it in email thread

**Pull request merged**